desk/tests/test_console_commands.py

```diff
@@ -1,14 +1,52 @@
+import os
 import pytest
 import numpy as np
 from astropy.table import Table
 from desk import console_commands
 from desk.set_up import config
+from PIL import Image
+
+
+def assert_images_equal(image_1: str, image_2: str):
+    # Function care of Jennifer Helsby @redshiftzero
+    img1 = Image.open(image_1)
+    img2 = Image.open(image_2)
+
+    # Convert to same mode and size for comparison
+    img2 = img2.convert(img1.mode)
+    img2 = img2.resize(img1.size)
+
+    sum_sq_diff = np.sum(
+        (np.asarray(img1).astype("float") - np.asarray(img2).astype("float")) ** 2
+    )
+
+    if sum_sq_diff == 0:
+        # Images are exactly the same
+        pass
+    else:
+        normalized_sum_sq_diff = sum_sq_diff / np.sqrt(sum_sq_diff)
+        assert normalized_sum_sq_diff < 0.001
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/s-goldman/Dusty-Evolved-Star-Kit/pullRequest?prid=5896643)